### PR TITLE
Qualify mysql 8.0 reserved keyword rank in PHP code

### PR DESCRIPTION
**Pull request by tussosedan**
Following up on #4522, this prepares the SQL statements in the PHP code for upgrade to mysql 8.0.

### CODE REVIEW

File: webroot/results/includes/regions_get_current_records.php [hunk 1]
```diff
@@ -7,21 +7,24 @@
...
       event.cellName   eventCellName,
-                       format,
+                       event.format,
       country.name     countryName,
       competition.cellName competitionName,
-                       rank, year, month, day
+                       event.rank,
+                       competition.year,
+                       competition.month,
+                       competition.day
```
Comment (jonatanklosko) [inline webroot/results/includes/regions_get_current_records.php]: Nit: just to follow the convention used here, please align these to the left.

Comment (tussosedan) [reply webroot/results/includes/regions_get_current_records.php]: I actually tried following the convention here for columns with aliases vs the original column names
image

Comment (jonatanklosko) [reply webroot/results/includes/regions_get_current_records.php]: Ah! That looks weird, but let's leave it like that then ^^

**Comment by jfly** [comment]
LGTM! I'll go ahead and deploy this to staging now.

Thanks, @tussosedan!

**Comment by jfly** [comment]
Err, whoops, I merged this to master. I guess we're going straight to prod =)

**Pull request merged**